### PR TITLE
Delay

### PR DESCRIPTION
**Pull request by danielmeans**
Added two bash scripts:
1. delay.sh: simulate geographically distributed network delays between a network of nodes
2. delay_stop.sh: remove delays

### CODE REVIEW

File: delay.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#Program to simulate network delay between nodes
+# Run as follows: "./tc_new.sh <instance_number>
```
Comment (estensen) [inline delay.sh]: It's not run as `./delay.sh <instance_number>`?

File: delay.sh [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+
+#Create IP Delay Tables
+IP_list=( 128.111.84.146 128.111.84.157 128.111.84.160 128.111.84.161 128.111.84.165 )
+
+let "instance_no = $1 - 1"
+
+declare -A DTable
+declare -a Delays
+Delays=(1 22 65 136 185 22 1 88 125 182 65 88 1 73 121 136 125 73 1 185 185 182 121 185 1)
```
Comment (estensen) [inline delay.sh]: Are these random or between actual AWS datacenters?

Comment (danielmeans) [reply delay.sh]: These are the actual delays between AWS datacenters in California, Oregon, Virginia, Ireland and Sao Paolo.

File: delay.sh (hunk 1 above)
Comment (danielmeans) [reply delay.sh]: Good catch, changed the file name.

**Pull request merged**